src/nni_manager/rest_server/restValidationSchemas.ts

```diff
@@ -140,6 +140,15 @@ export namespace ValidationSchemas {
                 }),
                 uploadRetryCount: joi.number().min(1)
             }),
+            dlts_config: joi.object({ // eslint-disable-line @typescript-eslint/camelcase
+                dashboard: joi.string().min(1),
+              
+                cluster: joi.string().min(1),
+                team: joi.string().min(1),
+              
+                email: joi.string().min(1),
+                password: joi.string().min(1)
```

Comment:
not very familiar to dlts, is it proper to ask for password from users?

Reply:
https://github.com/Gerhut/nni/blob/dlts-integration/src/nni_manager/training_service/dlts/dltsTrainingService.ts#L253-L264

These fields are all optional and could be retrieved if running in DLTS job.